src/07-registers/optimization.md

```diff
@@ -183,32 +145,77 @@ fn main() -> ! {
 
     loop {}
 }
+
 ```
 
-If we look at the disassembly of this new program compiled in release mode:
+Generate `release.txt` using with `--release` mode.
 
 ``` console
-$ cargo objdump --bin registers --release -- -d --no-show-raw-insn --print-imm-hex --source
-registers:      file format ELF32-arm-little
+cargo objdump --release --bin registers -- -d --no-show-raw-insn --print-imm-hex --source > release.txt
```

Comment:
```suggestion
$ cargo objdump --release --bin registers -- -d --no-show-raw-insn --print-imm-hex --source > release.txt
```
For consistency

Reply:
I purposely didn't add the `$` as this makes the `copy` button more useful.